### ISSUE TITLE
Clarify rule two

### ISSUE THREAD

**Issue by Taneb** [haskell/pvp#40]
Rule 2 currently says:

> _Non-breaking change_. Otherwise, if only new bindings, types, classes, non-orphan instances or modules (but see below) were added to the interface, then A.B **MAY** remain the same but the new C **MUST** be greater than the old C. Note that modifying imports or depending on a newer version of another package may cause extra non-orphan instances to be exported and thus force a minor version change.

There's an ambiguity here. It could be read as requiring C to be increased only when A.B remains the same (which I believe to be intended), or it could be read as requiring C to be increased regardless of whether A.B remains the same. Could this wording be made more clear?

**Comment by bergmark** [comment]
I agree, if the initial version is v0=A.B.C then v1 must have an increment in at least one of those components

**Comment by juhp** [comment]
I suppose the intended meaning is "but _then_ ".
